.github/workflows/dedupe.yaml

```diff
@@ -0,0 +1,54 @@
+name: Deduplicate Dependabot
+
+on:
+  push:
+    branches:
+      - dependabot/npm_and_yarn/*
+
+env:
+  NODE_VERSION: 16
+  NODE_OPTIONS: --max_old_space_size=6144
+
+permissions:
+  contents: write
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  dedupe:
+    name: Deduplicate dependencies
+    # Only trigger on initial commit from dependabot
+    if: github.actor == 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate app token
+        # Use a GitHub app to checkout and commit in order to re-trigger the CI workflow
+        # (because actions with GITHUB_TOKEN do not trigger new events)
+        id: generate_token
+        uses: tibdex/github-app-token@v1.7.0
+        with:
+          app_id: ${{ secrets.HA_COMMITTER_APP_ID }}
+          private_key: ${{ secrets.HA_COMMITTER_PRIVATE_KEY }}
```

Comment:
@balloob I'm about to transfer ownership of a simple GH app with content permissions.  Could you:
    
    1. Add a couple secrets for the App ID (not client ID) and private key with these names?  You'll have to generate a new private key and delete the existing one.
    2. Install the app in the frontend repo?